version/version.go

```diff
@@ -0,0 +1,13 @@
+package version
+
+import "strconv"
```

Comment:
```suggestion
import "fmt"
```

Add import for `fmt.Sprintf`